blocks/init/src/Blocks/components/social-links/components/social-links-editor.js

```diff
@@ -24,31 +23,26 @@ export const SocialLinksEditor = (attributes) => {
 		selector(additionalClass, additionalClass),
 	]);
 
-	const SocialItem = (props) => {
-		const {
-			href,
-			icon,
-			title,
-		} = props;
+	const socialLinksItemClass = selector(componentClass, componentClass, 'item');
+	const socialLinksLinkClass = selector(componentClass, componentClass, 'link');
 
-		return (
-			<li className={`${componentClass}__item`}>
-				<a className={`${componentClass}__link`} href={href} title={title} dangerouslySetInnerHTML={{ __html: icons[icon] }} target="_blank" rel="nofollow noreferrer noopener"></a>
-			</li>
-		);
-	};
+	if (!socialLinksUse) {
+		return null;
+	}
 
 	return (
-		<>
-			{socialLinksUse &&
-				<ul className={socialLinksClass}>
-					{socialLinksItems.map((element, index) => {
-						return (
-							<SocialItem {...element} key={index} />
-						);
-					})}
-				</ul>
-			}
-		</>
+		<ul className={socialLinksClass}>
+			{socialLinksItems.map(({ href, icon, title }, index) => (
+				<li className={socialLinksItemClass} key={index}>
+					<a
+						className={socialLinksLinkClass}
+						href={href}
+						title={title}
+						dangerouslySetInnerHTML={{ __html: manifest.icons[icon] }}
+						target="_blank" rel="nofollow noreferrer noopener"
```

Comment:
Do we want this by default? I think there is a legit use case for these attributes, but sometimes they could hurt the SEO iirc

Reply:
@infinum/wordpress-team chime in, please

Reply:
`nofollow` should generally only be used for:
- Ads
- Paid links
- If you're linking to something shady xD

So here I don't think we should have a `nofollow` a branded social link is a perfectly valid link IMO.

Reply:
`noreferrer noopener` is a security-issue fix for links opening in new tab so this should naturally stay.

Reply:
Fixed!